pygmt/datasets/earth_relief.py

```diff
@@ -50,6 +50,13 @@ def load_earth_relief(resolution="01d", region=None, registration=None):
         a pixel-registered grid is returned unless only the
         gridline-registered grid is available.
 
+    use_srtm : bool
+        By default, the land-only SRTM tiles from NASA are used to generate the
+        ``'03s'`` and ``'01s'`` grids, and the missing ocean values are filled
+        by up-sampling the SRTM15+V2.1 tiles which have a resolution of 15
+        arc-second (i.e., ``'15s'``). If True, will only load the original
```

Comment:
@seisman I added quotes for `'15s'`, but the region argument does use quotes:
https://github.com/GenericMappingTools/pygmt/blob/afe9293d8d6705587be9982909820d9ebf5e75eb/pygmt/datasets/earth_relief.py#L44-L45

Shall we use quotes for all the resolutions? or it does not matter?

Reply:
Best to be consistent and add quotes to all arguments, i.e. go with ` ``'05m'`` `.

Reply:
Shall I revise it to be ``'05m'`` in this PR? or in another PR for this typo?

Reply:
If we want to be consistent with <code>``'05m'``</code>, we have to make more changes to other files, for example, 

>      registration : str
>         Grid registration type. Either ``pixel`` for pixel registration or
>         ``gridline`` for gridline registration. Default is ``None``, where
>         a pixel-registered grid is returned unless only the
>         gridline-registered grid is available.

<code>``pixel``</code> should be <code>``'pixel'``</code>.

Reply:
Just checked other projects, numpy (https://numpy.org/doc/stable/reference/generated/numpy.datetime_as_string.html#numpy.datetime_as_string), pandas (https://pandas.pydata.org/docs/reference/api/pandas.read_csv.html) and matplotlib (https://matplotlib.org/stable/api/legend_api.html) use uses `'string'`, not ` ``'string'`` `.

Reply:
How about discussing this in another issue? We may decide which way is better and have a to-do list to revise all the documentation.

Reply:
Yes, please open a separate issue so that we can better discuss it. In this PR, we can use ` ``'05m'`` ` for consistency and decide what we want later.